ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityActivityBlock.ts

```diff
@@ -0,0 +1,21 @@
+import { ActivityBlock } from './activityBlock';
+import { BlockType } from './blockType';
+import { ActivityInstance } from 'ddp-sdk';
+
+enum ActivityRenderHintType {
+    Embedded = 'EMBEDDED',
+    Modal = 'MODAL'
+}
```

Comment:
We have been giving this type of definitions their own files. We should keep with that approach.

Reply:
Got it, fixed in a8003463771fa957fff2a8cefafc757b2b74ee3d

Reply:
In addition, what are your thoughts about a structuring files into directories ? 
E.g. `ddp-sdk\src\lib\models\activity` and `ddp-sdk\src\lib\models`. Each of them has more than 50 files, it is quite confusing.

My suggestion is to sort those files into some sub-folders, either '**_by features_**' or '**_by file types_**' (../enums, ../interfaces)

Reply:
I am personally not bothered with the large folder as I tend not to click/navigate through folders. I generally look for classes and files by typing their name or part of name, and would continue to do that regardless of folder arrangement.
But if you feel strongly that we should split "model" into more subfolders, sure. I am not opposed to idea.
Now, I do have a strong dislike for separating enums from interfaces (I see them as serving same purpose: defining the shape of objects and data), but if you can come up with a feature-based folder scheme that make sense, go for it.

Reply:
Ok, let me think about it.